.gitmodules

```diff
@@ -19,6 +19,6 @@
 [submodule "cli/bin/vendor/urllib3"]
 	path = cli/bin/vendor/urllib3
 	url = https://github.com/urllib3/urllib3
-[submodule "cli/bin/vendor/cryptography"]
-	path = cli/bin/vendor/cryptography
-	url = https://github.com/pyca/cryptography
+[submodule "cli/bin/vendor/lark-parser"]
```

Comment:
Need to keep the cryptography module as it is not present by default in python3.x, so that our codebase is compatible with both the versions.

Reply:
we're not using cryptography anymore, as #317 removes it in favor of `pyes`